ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main__mce.yaml

```diff
@@ -22,12 +22,12 @@ images:
   to: assisted-image-service
 promotion:
   to:
-  - name: "2.5"
+  - name: "2.6"
     namespace: stolostron
 releases:
   latest:
     integration:
-      name: "4.13"
+      name: "4.16"
```

Comment:
This is unrelated right ?

Reply:
yes, unrelated, just to avoid some scripts in dptp to detect this file and do something with it.